deps/rabbit/test/config_schema_SUITE_data/rabbit.snippets

```diff
@@ -898,6 +898,21 @@ credential_validator.regexp = ^abc\\d+",
      ]}],
    []},
 
+  %%
+  %% Runtime parameters
+  %%
+
+  {runtime_parameters_limits,
+  "
+   runtime_parameters.limits.b = 2
```

Comment:
Should we use more realistic parameter names (e.g. `shovel`) here so that it's easier to understand how to use this feature?